### PR TITLE
Update Device Agent docs for 3.x release

### PR DESCRIPTION
**Pull request by knolleary**
Updates the docs for the agent around supported versions of Node.js.

### CODE REVIEW

File: docs/device-agent/install.md [hunk 1]
```diff
@@ -125,6 +125,12 @@ For backwards compatibility we will continue to publish to both the old and
 new locations for a period of time, but we strongly encourage users to update to the
 new package to ensure you continue to receive the latest updates.
 
+### Upgrading to Device Agent 3.x
+
+Version 3.x of the Device Agent requires Node.js 18 as a minimum; older versions of Node.js are no longer supported.
+
+The `latest` tagged Docker image is now based on Node.js 18.
+
```
Comment (hardillb) [inline docs/device-agent/install.md:133]: No comment about installing the older version? Or just trying to keep it simple?

**Review by hardillb — APPROVED** [review]
Just the question about not referencing the 2.x version.

If that's fine feel free to merge

File: docs/device-agent/install.md (hunk 1 above)
Comment (knolleary) [reply docs/device-agent/install.md:133]: I bounced around on it... will have a second look.

Comment (knolleary) [reply docs/device-agent/install.md:133]: @hardillb added a comment

**Review by hardillb — APPROVED** [review]

**Pull request merged**